bundles/org.openhab.binding.bluetooth.bluez/src/main/resources/ESH-INF/thing/bluez.xml

```diff
@@ -20,18 +20,18 @@
 				<advanced>true</advanced>
 				<default>false</default>
 			</parameter>
-			<parameter name="inactiveDeviceCleanupIntervalSecs" type="integer" min="1">
-				<label>Device Cleanup Interval</label>
-				<description>How often device cleanup is performed in seconds</description>
-				<advanced>true</advanced>
-				<default>60</default>
-			</parameter>
-			<parameter name="inactiveDeviceCleanupThresholdSecs" type="integer" min="1">
-				<label>Device Cleanup Threshold</label>
-				<description>The number of seconds a device can remain radio silent before it is eligible for cleanup</description>
-				<advanced>true</advanced>
-				<default>300</default>
-			</parameter>
+			<parameter name="inactiveDeviceCleanupInterval" type="integer" min="1" unit="s">
```

Comment:
formatting looks wrong

Reply:
Where?

Reply:
Ah, you must be looking at the incremental commits. The formatting was fixed in the followup merge commit.

Reply:
Ah, you must be looking at the incremental commits. The formatting was fixed in the followup merge commit.